### ISSUE TITLE
License Renewal

### ISSUE THREAD

**Issue by ScottB-Ronco** [hougaard/Simple-Object-Designer#209]
My license is coming up for renewal in a couple of months and I was on you website at hougaard.com and noticed that two things that were mentioned. 

On FAQ page for the question "How do I renew my SOD license?" you stated that you now support subscriptions with a note that if we purchased SOD before March 2023 it will be the last time you will need to purchase a new license key.

If I go to the section to buy a license for SOD it has a note saying "To existing customers: We have switched to a subscription system instead of single-use license keys. So this time, you need to purchase a new license key for your renewal; it will be extended automatically next time." If I click on the more details it goes to a page not found

With that I would like some clarification on how license renewal will work for me as I've be using SOD since early days of v1 and I've purchased it well before March 2023.

**Comment by hougaard** [comment]
Hi Scott,

We have switched over to a new (behind the scene) way to manage license keys.

The old system was like a scratch ticket, you purchased a license key and popped that into the app. When it expired, you had to purchase a new license key and apply that. We had no way of extending the active period of the license key.

With the new method, you purchase a "subscription license key" (looks just like the others) and pop that in. Then as long as your subscription is active, your app will work.

I see that your license will expire in July, so until then, you just keep using the license you have.

**Comment by ScottB-Ronco** [comment]
Yes I understand that under the new system it is now subscription based.

However in one area of your site it says if I purchased before March 2023 it is the last time you need to buy a new license key. I guess this means that once I buy a new one in July I will be switched to the new subscription key?

Also will the cost still be a flat rate for the year or have you switched to monthly or monthly based on user count like so many other apps do?

**Comment by hougaard** [comment]
Yes, you cannot buy the old keys anymore.

Still a yearly flat rate.
